app.rb

```diff
@@ -21,6 +22,11 @@ class App < Sinatra::Base
     ''
   end
 
+  post '/user-signup/sms-notification' do
+    SmsSignup.new(user_model: User.new).execute(contact: params[:source])
```

Comment:
If this raised an exception would that be returned to FireText?

Reply:
This is what Sinatra returns to the client a 500.

```
< HTTP/1.1 500 Internal Server Error
< Content-Type: text/html;charset=utf-8
< X-XSS-Protection: 1; mode=block
< X-Content-Type-Options: nosniff
< X-Frame-Options: SAMEORIGIN
< Content-Length: 30
< 
* Connection #0 to host localhost left intact
<h1>Internal Server Error</h1>% 
```

We'd get an entry inside of Sentry.